src/Raven.Client/Exceptions/Commercial/LimitType.cs

```diff
@@ -62,6 +62,9 @@ public enum LimitType
         EncryptedBackup,
 
         [Description("Additional Assemblies from NuGet")]
-        AdditionalAssembliesFromNuGet
+        AdditionalAssembliesFromNuGet,
+        
+        [Description("Endpoints Monitoring")]
```

Comment:
Shouldn't this be the other way around? Monitoring Endpoints?

Reply:
changed